docs/user/resources/01-telemetry.md

```diff
@@ -88,7 +88,7 @@ The state of the log components is determined by the status condition of type `L
 | True             | FluentBitDaemonSetReady    | Fluent Bit DaemonSet is ready                   |
 | False            | ReferencedSecretMissing    | One or more referenced Secrets are missing      |
 | False            | FluentBitDaemonSetNotReady | Fluent Bit DaemonSet is not ready               |
-| False            | ResourceBlocksDeletion     | The deletion of the module is blocked. To unblock the deletion, delete the following resources: LogPipelines (resource-1, resource-2,...), LogParsers (resource-1, resource-2,...) |
```

Comment:
Should we already remove this part from the documentation?
It is not motivating users to use the LogParsers, it just shows the actual message that will pop up in case that the deprecated LogParser is used.